.eslintrc

```diff
@@ -11,5 +11,11 @@
     "node": true,
     "es6": true,
     "mocha": true
+  },
+  "rules": {
+    "@typescript-eslint/no-unused-vars": ["error", {
+      "args": "none"
```

Comment:
Unused arguments are declared in a lot of places, and it could be argued they convey information about APIs being overwritten, or opportunities for extension e.g.

https://github.com/brianc/node-postgres/blob/a24a24dea19481fefb84ee42e7e5cbe0f2eb06b4/packages/pg/test/unit/connection/error-tests.js#L67

https://github.com/brianc/node-postgres/blob/a24a24dea19481fefb84ee42e7e5cbe0f2eb06b4/packages/pg/lib/query.js#L122